README.md

```diff
@@ -144,26 +148,23 @@ VBR.out.viscous.HZK2011.eta_total % composite steady state creep viscosity
 
 The VBR Calculator nominally works in GNU Octave, but you may find that you need to install some packages.
 
-https://octave.sourceforge.io/io/index.html
-https://octave.sourceforge.io/statistics/index.html
-https://octave.org/doc/interpreter/Installing-and-Removing-Packages.html
+https://octave.sourceforge.io/io/index.html https://octave.sourceforge.io/statistics/index.html https://octave.org/doc/interpreter/Installing-and-Removing-Packages.html
 
 # Getting Help
 
-See the full documentation page on [Contributing and Getting Help](https://vbr-calc.github.io/vbr/contrib/contributing/)
-for ways to reach out or [post a github issue](https://github.com/vbr-calc/vbr/issues/new)!
+See the full documentation page on [Contributing and Getting Help](https://vbr-calc.github.io/vbr/contrib/contributing/) for ways to reach out or [post a github issue](https://github.com/vbr-calc/vbr/issues/new)!
 
 # Contributing to the VBRc
 
 The VBRc is open to community contributions! New methods, new examples, documentation fixes, bug fixes!
 
 We follow a typical open source workflow. To submit new features:
 
-* create your fork of the VBRc repo
-* checkout a new branch
-* do work on your new branch
-* push those changes to your fork on github
-* submit a pull request back to the main VBRc repo
+- create your fork of the VBRc repo
+- checkout a new branch
+- do work on your new branch
+- push those changes to your fork on github
+- submit a pull request back to the main VBRc repo
```

Comment:
I'm guessing your IDE auto-changed this. I'd rather keep the * syntax for the unorded list though.
```suggestion
* create your fork of the VBRc repo
* checkout a new branch
* do work on your new branch
* push those changes to your fork on github
* submit a pull request back to the main VBRc repo
```